CONTRIBUTING.rst

```diff
@@ -70,8 +80,8 @@ that particular check was disabled and it MUST be scoped as narrowly as
 possible. i.e. Don't disable linting on an entire class or method
 when disabling the check for a single statement will do.
 
-This repository uses [pre-commit](https://pre-commit.com) to check and enforce code style. It uses
-[Black](https://github.com/psf/black) to reformat the Python code and [Flake8](http://flake8.pycqa.org) to check it
+This repository uses `pre-commit <https://pre-commit.com />` to check and enforce code style. It uses
+`Black <https://github.com/psf/black />` to reformat the Python code and `Flake8 <http://flake8.pycqa.org />` to check it
```

Comment:
Looks kind of "funky" when viewed in [GH](https://github.com/project-koku/koku/blob/add_dco/CONTRIBUTING.rst):
<img width="983" alt="image" src="https://user-images.githubusercontent.com/29379759/78389518-d6672800-75b0-11ea-83e4-0027d7781db9.png">

Reply:
Fixed!